.eslintrc

```diff
@@ -12,6 +12,7 @@
     "node": true
   },
   "rules": {
+    "max-len": ["error", {"code": 90}],
```

Comment:
I would not change the general eslint rule of `max-len`. Next day someone else comes along and needs to widen it to a 100 and so on. It's rather better to make an exception in the file. Please see the suggestion down below.